jujugui/static/gui/src/app/utils/environment-change-set.js

```diff
@@ -638,6 +661,11 @@ YUI.add('environment-change-set', function(Y) {
               delete config[key];
             }
           });
+          // Check if we have any resources, if we do, then add them.
+          const resourceIds = ghostService.get('resourceIds');
```

Comment:
Having one call executed after the other without the need for storing the pending ids in a state would have been better, but I understand that this may require changes in the ecs we don't want to do now. Are we sure this little state here will be cleaned up later? I guess the whole ghost service will be destroyed after firing the ecs.